src/battle_script_commands.c

```diff
@@ -3489,11 +3489,10 @@ static void Cmd_getexp(void)
     }
 }
 
-#ifdef NONMATCHING
 static void Cmd_unknown_24(void)
 {
-    u16 HP_count = 0;
     s32 i;
+    u16 HP_count = 0;
```

Comment:
Is switching the declaration order necessary to match?

Reply:
I fixed it, so